README.md

```diff
@@ -9,9 +9,10 @@ This project involves creating a responsive website for Pet Care company that fe
 ## Setup and Usage
 
 1. Clone the repository to your local machine.
-2. Run `docker-compose up -d` OR `docker compose up -d`.
-3. Visit http://localhost:8081/home.html
-4. You are all set up to edit the webpage! Enjoy 😄
+2. On the terminal `cd public`
+3. Run `docker-compose up -d` OR `docker compose up -d`.
```

Comment:
Why would I run docker from the public folder? 
Is compose.yaml a file that should be public to an end user?